lib/recurly/purchase.php

```diff
@@ -54,11 +54,43 @@ public static function preview($purchase, $client = null) {
    * @param Recurly_Client $client Optional client for the request, useful for mocking the client
    * @return object Recurly_InvoiceCollection
    * @throws Recurly_Error
+   *
+   * Note: To use this endpoint, you may have to contact Recurly support to have it enabled on your subdomain.
    */
   public static function authorize($purchase, $client = null) {
     return Recurly_Base::_post('/purchases/authorize', $purchase->xml(), $client);
   }
 
+  /**
+   *
+   * Capture an open Authorization request
+   *
+   * @param $purchase Recurly_Purchase Our purchase data.
```

Comment:
The parameter in the docs should be `$transactionUUID` instead of `$purchase`. It's a string, not a `Recurly_Purchase`. Same with the cancel method docs. It might be nice to spell out exactly where the developer can find the transaction uuid in the invoice collection returned from the Create Purchase endpoint. Your test gives a pretty decent code example of how to yank the UUID, maybe include that in the PHPdoc?